repos/system_upgrade/el7toel8/actors/authselectscanner/libraries/library.py

```diff
@@ -0,0 +1,105 @@
+import errno
+import re
+
+from leapp.libraries.stdlib import call
+from subprocess import CalledProcessError
+
+
+def read_file(file):
+    """
+        Read file contents. Return empty string if the file does not exist.
+    """
+    try:
+        with open(file, "r") as f:
+            return f.read()
+    except IOError as err:
+        if err.errno == errno.ENOENT:
+            return ""
+        else:
+            raise 
+
+
+def is_service_enabled(service):
+    """
+        Return true if @service is enabled with systemd, false otherwise.
+    """
+    try:
+        call(
+            ["/usr/bin/systemctl", "is-enabled", service + ".service"],
+        )
+    except (OSError, CalledProcessError):
+        return False
+
+    return True
+
+
+class Authconfig(object):
```

Comment:
Correct me if I'm wrong, please
It look like classes Authconfig and DConf do the same config parsing (if the option is enabled, the only difference is the actual value - yes vs true - of the truth value). Can the code be generalized and reused then?

Reply:
Done.